test/verify/check-selinux

```diff
@@ -192,7 +193,7 @@ class TestSelinux(MachineCase):
             # fallback for older Chrome releases
             b.grant_permissions("clipboardRead", "clipboardWrite")
 
-        b.wait_visible(".pf-c-data-list__cell:contains(Allow zebra to write config)")
+        b.wait_visible(".pf-c-data-list__cell:contains(Allow zebra)")
```

Comment:
Should this also be "Allow zebra.*write.*config" like above for .modifications-table?

Reply:
Because we use the `:contains()` selector, we can't put a regexp here.

Reply:
Ouch, obviously.